packages/sdk-components-react/src/time.tsx

```diff
@@ -327,6 +327,23 @@ const timeStyleOrUndefined = (
   return undefined;
 };
 
+export const parseDate = (datetimeString: string) => {
+  if (datetimeString === "") {
+    return;
+  }
+  const isNumber = /^\d+$/.test(datetimeString);
+  let timestamp = Number(datetimeString);
+  if (isNumber && datetimeString.length === 10) {
+    timestamp = timestamp * 1000;
+  }
+  const date = new Date(isNumber ? timestamp : datetimeString);
+  const isValid = false === Number.isNaN(date.getTime());
+
+  if (isValid) {
+    return date;
+  }
+};
+
```

Comment:
IMO better to check that Date isValid first and only then try any other formats.
Or explain why we check non std date formats first

Reply:
Why as Date with 10 digits string is NaN

Reply:
`new Date("1000000000")` <- INVALID